### PR TITLE
Add new gRPC based ann-benchmark subcommand

### PR DESCRIPTION
**Pull request by trengrj**

### CODE REVIEW

File: benchmarker/cmd/ann_benchmark.go [hunk 1]
```diff
@@ -0,0 +1,751 @@
...
+	EfConstruction   int     `json:"efConstruction"`
+	MaxConnections   int     `json:"maxConnections"`
+	Mean             float64 `json:"meanLatency"`
+	P99Latency       float64 `json:"p99Latency"`
+	QueriesPerSecond float64 `json:"qps"`
+	Shards           int     `json:"shards"`
+	Parallelization  int     `json:"parallelization"`
+	Limit            int     `json:"limit"`
+	ImportTime       float64 `json:"importTime"`
+	RunID            string  `json:"run_id"`
```
Comment (asdine) [inline benchmarker/cmd/ann_benchmark.go:47]: Do we have both camelCase and snake case or is it a typo?

File: benchmarker/cmd/benchmark_run.go [hunk 2]
```diff
@@ -14,10 +15,123 @@ import (
...
+		if err != nil {
+			fmt.Printf("ERROR: %v\n", err)
+			continue
+		}
+		took := time.Since(before)
+		bytes, _ := ioutil.ReadAll(res.Body)
+		res.Body.Close()
+		var result map[string]interface{}
+		if err := json.Unmarshal(bytes, &result); err != nil {
+			fmt.Printf("JSON error: %v\n", err)
```
Comment (asdine) [inline benchmarker/cmd/benchmark_run.go:63]: If we ignore the error here, line 66 and 74 will panic because result is not instantiated.

**Review by asdine — APPROVED** [review]
LGTM

File: benchmarker/cmd/ann_benchmark.go (hunk 1 above)
Comment (trengrj) [reply benchmarker/cmd/ann_benchmark.go:47]: @asdine good point. I kept the inconsistent casing of `run_id` / `maxConnections` etc to match the json format of the original python chaos benchmarks.  I'd like to keep the same keys so we can plot both results.

**Pull request merged**